remappings.txt

```diff
@@ -9,3 +9,6 @@
 @lm_pc/=src/modules/logicModule/paymentClient/
 @aut/=src/modules/authorizer/
 @modLib/=src/modules/lib/
+@mock/=test/mocks/
```

Comment:
Would go with the whole name of the directory, i.e. `mocks` instead of `mock` to reduce confusion. Same for tool